README.md

```diff
@@ -30,17 +62,21 @@ python setup.py install
 ```
 It should be noted that this is a working branch and is subject to change.
 
-# Using with pysat
+# Examples
 
-The instrument modules are portable and designed to be run like any pysat instrument.
+The instrument modules are portable and designed to be run like any pysat
+instrument.
 
 ```
 import pysat
 from pysatSpaceWeather.instruments import sw_dst
 
 dst = pysat.Instrument(inst_module=sw_dst)
 ```
-Another way to use the instruments in an external repository is to register the instruments.  This only needs to be done the first time you load an instrument.  Afterward, pysat will identify them using the `platform` and `name` keywords.
+
+Another way to use the instruments in an external repository is to register the
+instruments.  This only needs to be done the first time you load an instrument.
+Afterward, pysat will identify them using the `platform` and `name` keywords.
 
 ```
 import pysat
```

Comment:
`import pysatSpaceWeather`

Reply:
Actually, following the example above we don't need any more imports :P

Reply:
Oh, I see.  Registry needs the module imported.